.github/workflows/ci.yml

```diff
@@ -48,3 +48,32 @@ jobs:
         if: ${{ matrix.coverage }}
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
+  test-legacy:
+    name: Test - ubuntu-latest - Node v6.x, Webpack 4
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install dependencies
+        run: yarn
+        env:
+          YARN_NODE_LINKER: node-modules
+      - name: Install webpack 4
+        run: yarn add -D webpack@4
+        env:
+          YARN_NODE_LINKER: node-modules
+      - name: Build babel-loader
+        run: yarn run build
+        env:
+          YARN_NODE_LINKER: node-modules
```

Comment:
The env `YARN_NODE_LINKER ` can be moved to the job scope: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#env